src/main/java/org/mozilla/msrp/platform/vertical/video/YoutubeRepository.kt

```diff
@@ -67,4 +70,20 @@ class YoutubeRepository @Inject constructor(private val youtube: YouTube) {
             emptyList()
         }
     }
+
+    private fun String.compact(): String {
+        val duration = Duration.parse(this)
```

Comment:
Suggest to catch DateTimeParseException for better debugging

Reply:
Added